src/app/routes/cpsAsset/utils/getAssetType.test.js

```diff
@@ -0,0 +1,25 @@
+import getAssetType from './getAssetType';
```

Comment:
I like the convention of separating our imports 👍 
```suggestion
// components to test
import getAssetType from './getAssetType';
```

Reply:
:) Ive started doing, I find it hard to read through the imports when we sometimes have lists of like 20+